src/vs/editor/contrib/multicursor/multicursor.ts

```diff
@@ -1050,6 +1050,10 @@ export class SelectionHighlighter extends Disposable implements IEditorContribut
 		description: 'selection-highlight-overview',
 		stickiness: TrackedRangeStickiness.NeverGrowsWhenTypingAtEdges,
 		className: 'selectionHighlight',
+		minimap: {
+			color: themeColorFromId(minimapFindMatch),
```

Comment:
I suggest that we create a new color e.g. `minimalSelectionHighlight` and not reuse `minimapFindMatch`